src/NzbDrone.Core/Parser/QualityParser.cs

```diff
@@ -324,83 +332,131 @@ public static QualityModel ParseQualityName(string name)
 
             if (resolution == Resolution.R2160p)
             {
+                result.ResolutionDetectionSource = QualityDetectionSource.Name;
                 result.Quality = remuxMatch ? Quality.Bluray2160pRemux : Quality.HDTV2160p;
+
                 return result;
             }
 
             if (resolution == Resolution.R1080p)
             {
+                result.ResolutionDetectionSource = QualityDetectionSource.Name;
                 result.Quality = remuxMatch ? Quality.Bluray1080pRemux : Quality.HDTV1080p;
+                
                 return result;
             }
 
             if (resolution == Resolution.R720p)
             {
+                result.ResolutionDetectionSource = QualityDetectionSource.Name;
                 result.Quality = Quality.HDTV720p;
+                
                 return result;
             }
 
             if (resolution == Resolution.R360P || resolution == Resolution.R480P)
             {
+                result.ResolutionDetectionSource = QualityDetectionSource.Name;
                 result.Quality = Quality.SDTV;
+                
                 return result;
             }
 
             if (codecRegex.Groups["x264"].Success)
             {
                 result.Quality = Quality.SDTV;
+                
                 return result;
             }
 
             if (normalizedName.Contains("848x480"))
             {
+                result.ResolutionDetectionSource = QualityDetectionSource.Name;
+                
                 if (normalizedName.Contains("dvd"))
                 {
+                    result.SourceDetectionSource = QualityDetectionSource.Name;
                     result.Quality = Quality.DVD;
                 }
+                else if (normalizedName.Contains("bluray"))
```

Comment:
ContainsIgnoreCase (I removed the ToLower on normalizedName earlier today to allow a regex to do a case-sensitive match)